packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -185,11 +193,9 @@ export abstract class TelemetryLogger implements ITelemetryLogger {
         this.sendTelemetryEventCore(perfEvent, error);
     }
 
-    protected prepareEvent(event: ITelemetryBaseEvent): ITelemetryBaseEvent {
+    protected prepareEvent(event: ITelemetryBaseEventExt): ITelemetryBaseEventExt {
```

Comment:
Side note - Not sure why this function isn't used in this base class... Not related to this PR but maybe a small fix to discuss later.